doc/manual/manual.tex

```diff
@@ -10042,6 +10042,31 @@ \subsubsection{Brittle thrust wedges benchmark}
 \end{figure}
 
 
+\subsection{Setups for teaching}
+\label{sec:cookbooks-teaching}
+
+Because \aspect{} is freely available, has an extensive documentation and can be applied to a variety of problems in geophysics, 
+it can be a useful tool for teaching geophysics in general or geodynamic modeling in particular. 
+In the following section, we will present a number of cookbooks that can be used for this purpose. 
+Many of them are modifications of existing cookbooks, but have been changed to run faster to be more suitable for
+running them in the classroom, or they include additional ideas for what parameters can be changed to learn more 
+about the physical behaviour that controls the model results. 
+
+\paragraph{Introduction to Geophysics}
+\label{sec:cookbooks-intro-geophysics}
+\textit{This section was contributed by Juliane Dannberg, based on the course ``Introduction to Geophysics'' at University of Florida.}
+
+The course is designed to teach general concepts of geophysics, and it includes the following cookbooks:
+\begin{enumerate}
+  \item \nameref{sec:cookbooks-running-a-model} 
+  \item \nameref{sec:cookbooks-heat-flow} 
+  \item ...
+\end{enumerate}
+
+\input{cookbooks/tutorial/doc/tutorial.tex}
```

Comment:
should these be put into the main section of cookbooks instead? This might be a good first cookbook or not?

Reply:
It may be a good input file for a first cookbook, but our current first cookbook (convection box) is almost the same, with less functionality, and that section contains a lot of helpful advice for first-time users. I thought the whole point about these teaching cookbooks is that the contributor doesn't have to write a lot of text in the manual, and instead there may be slides that accompany the cookbook. So I think this is somewhat different from the normal cookbooks.